code/code/misc/discord.cc

```diff
@@ -96,4 +90,27 @@ bool Discord::sendMessage(sstring channel, sstring msg)
     curl_slist_free_all(headers);
     curl_easy_cleanup(curl);
     return true;
+}
+
+void Discord::sendMessage(sstring channel, sstring msg, bool detach) 
+{
+    if (channel == "") {
+        // no channel configuration, so bail
+        return;
+    }
+
+    vlogf(LOG_MISC, format("Discord webhooks: '%s'")  % msg);
+
+    // we want to do this asynchronously so POST lag doesn't hang the mud
+    std::thread thread(
```

Comment:
I don't know, but pehaps you could say 
```cpp
std::thread thread(sendMessageAsync, channel, msg);
```
?

Reply:
this is better, I was overthinking it